.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
`.gitignore` this file, otherwise this will conflict with anyone else using VS Code.